input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+            <td style="width: 25%;">MedicationRequest.code</td>
+            <td style="width: 25%;">US Core uses an extensible binding to <a href="https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1010.4/expansion">Medication Clinical Drug</a>, which is not in AU medicines terminology.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html">AU Core Organization</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-organization.html">US Core Organization</a></td>
+            <td style="width: 25%;">Organization.active</td>
+            <td style="width: 25%;">US Core requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-patient.html">US Core Patient</a></td>
+            <td style="width: 25%;">Patient.identifier</td>
+            <td style="width: 25%;">US Core requires all identifiers to be valid (have system and value).</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Gender identity</td>
+            <td style="width: 25%;">US Core defines an US Core Gender Identity extension. AU Core uses the International Gender Identity extension.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Patient.telecom</td>
+            <td style="width: 25%;">US Core requires minimum of 1 system and value.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html">AU Core Practitioner</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitioner.html">US Core Practitioner</a></td>
+            <td style="width: 25%;">Practitioner.identifier</td>
+             <td style="width: 25%;">US Core requires minimum of 1, and requires all practitioner identifiers to have system and value.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html">AU Core PractitionerRole</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-practitionerrole.html">US Core PractitionerRole</a></td>
+            <td style="width: 25%;">PractitionerRole.telecom, PractitionerRole.endpoint</td>
+            <td style="width: 25%;">US Core has constraint pd-1, which mandates that resources SHALL have contact information or a reference to an Endpoint.</td>
```

Comment:
Rephrase "US Core requires telecom or endpoint to be present (pd-1)"

Reply:
Done